modules/runner/src/integrationTest/sql/function/generic/test_coalesce.test

```diff
@@ -45,7 +45,7 @@ statement ok
 INSERT INTO test3 VALUES (null, 22), (12, null), (13, 22)
 
 query T
-SELECT COALESCE(CAST(a AS VARCHAR), 11) FROM test3 ORDER BY a;
+SELECT COALESCE(CAST(a AS VARCHAR), 11) as b FROM test3 ORDER BY b;
```

Comment:
I believe the proper fix is to move 11 at the end

Reply:
done, also all others previous discussed issues are changed too. check plz ?